history/taskana-simplehistory-spring-test/src/test/java/pro/taskana/TaskHistoryEventControllerIntTest.java

```diff
@@ -129,10 +126,10 @@ public void testThrowsExceptionIfInvalidFilterIsUsed() {
           HttpMethod.GET,
           request,
           ParameterizedTypeReference.forType(TaskHistoryEventListResource.class));
-      fail();
+      fail("");
```

Comment:
Maybe we can add a fitting message to the fail like "Invalid filter is used" instead of an empty string. Also I'm not sure if we need this at all, can't we just use the assertThatThrownBy-pattern that we used in other cases? Pls consult with @benjamineckstein

Reply:
Yes, we should.